sw/device/lib/dif/README.md

```diff
@@ -4,7 +4,9 @@ A DIF is a "Device Interface Function". DIFs are low-level routines for
 accessing the hardware functionality directly, and are agnostic to the
 particular environment or context they are called from. The intention is that
 DIFs can be used during design verification, and during early silicon
-verification, and by the high-level driver software in production firmware.
+verification.
+
+DIFs are *not* intended to be used by production firmware.
```

Comment:
Some things we should clarify:
- Even though they are not production code, they are written to the same standards as production code.
- They are a valid starting point for writing production drivers, iow they're what a driver with no hardening and no narrow focus on specific operations would look like.

Reply:
I've updated the language to clarify.  PTAL.